docker-compose.yml

```
@@ -5,16 +5,18 @@ services:
     build: .
     ports:
       - "8080:8080"
+    env_file:
+      - .env
     environment:
-      - DB_USERNAME=${DB_NAME}
+      - DB_NAME=${DB_NAME}
```

DB_NAME c'est pas la même chose 👀 
```suggestion
      - DB_USERNAME=${DB_NAME}
```